spark/src/main/java/io/ddf/spark/content/SchemaHandler.java

```diff
@@ -102,7 +102,11 @@ public static Schema getSchemaFromDataFrame(DataFrame rdd) {
 
       for (Integer columnIndex : columnIndexes) {
         String colName = this.getDDF().getColumnName(columnIndex);
-        listLevelCountsWithName.put(colName, listLevelCounts.get(columnIndex));
+        if(listLevelCounts.get(columnIndex) != null) {
```

Comment:
just handle a case where the levels count is empty for the current column, it's a small bug not related to this PR